wasmbinding/query_plugin.go

```diff
@@ -110,6 +144,31 @@ func CustomQuerier(qp *QueryPlugin) func(ctx sdk.Context, request json.RawMessag
 	}
 }
 
+func NormalizeReponses(binding interface{}, bz []byte) ([]byte, error) {
+	// all values are proto message
+	message, ok := binding.(proto.Message)
+	if !ok {
+		return nil, wasmvmtypes.Unknown{}
+	}
+
+	// unmarshal binary into stargate response data structure
+	err := proto.Unmarshal(bz, message)
+	if err != nil {
+		return nil, wasmvmtypes.Unknown{}
+	}
+
+	// build new deterministic response
+	bz, err = proto.Marshal(message)
+	if err != nil {
+		return nil, wasmvmtypes.Unknown{}
+	}
+
+	// clear proto message
+	message.Reset()
```

Comment:
why is this necessary?

Reply:
@alexanderbez  tbh, I do not have full understanding why this is here left it here for now as I wanted to be careful of removing what I don't know it does. It was one of the chunks that I've forked from other repository that was using stargate query after enabling it. From my brief understanding, it seems that we are doing this because it was an Any value, and it would cached the old value. 

Looking at it again, I do feel that it might be safe to remove this line since we're not using the `message` varaible any more. Can you give opinions for this?

Reply:
mehh, honestly it's fine to leave it. I was just curious. I asked because `message` is created in the function and never leaves it.